pkg/devfile/parser/data/common/types.go

```diff
@@ -151,6 +151,13 @@ type Exec struct {
 
 	// Working directory where the command should be executed
 	WorkingDir string `json:"workingDir,omitempty" yaml:"workingDir,omitempty"`
+
+	// +optional
+	// Whether the command is capable to reload itself when source code changes.
+	// If set to `true` the command won't be restarted and it is expected to handle file changes on its own.
+	//
+	// Default value is `false`
```

Comment:
Can you please also specify in comment, what would default value do?

Reply:
firstly this description directly comes from upstream reference https://github.com/devfile/api/blob/master/schemas/devfile.json#L208

secondly `hotReload` is `bool`, `true` condition is sufficiently explained, false would be opposite of that.